x-pack/plugins/lens/public/visualizations/xy/types.ts

```diff
@@ -113,6 +113,7 @@ export interface XYAnnotationLayerConfig {
   layerId: string;
   layerType: 'annotations';
   annotations: EventAnnotationConfig[];
+  hide?: boolean;
```

Comment:
where are we using it? 🤔

Reply:
I think it does not belong to here. Will remove it. 👍 
```suggestion
```